### DISCUSSION TITLE
Set the angle of the bottom or top shell infill

### DISCUSSION THREAD

**Discussion by obfuscateddevices** [OrcaSlicer/OrcaSlicer#1359] [Q&A]
I have a carbon fiber textured plate and I think the texture will look best of the lines of the bottom shell are aligned to it but there doesn't appear to be a way to control the angle of the of the patterns. If I rotate the model the pattern is arbitrarily oriented. Isa there a way to do this in Orca or is this a feature request?

**Comment by julie777** [comment]
@obfuscateddevices 
Under strength you can change the angle of the infill, which includes the bottom surface. This shows the bottom layer.

![image](https://github.com/SoftFever/OrcaSlicer/assets/5090005/3970c942-b600-49ee-9962-e9d1592be391)
